.github/workflows/metrics.yml

```diff
@@ -75,6 +75,10 @@ jobs:
         if: matrix.backend == 'mysql'
         run: cargo +stable bench --manifest-path diesel_bench/Cargo.toml --no-default-features --features "mysql sqlx-bench sqlx/mysql tokio rustorm rustorm/with-mysql rustorm_dao rust_mysql futures sea-orm sea-orm/sqlx-mysql criterion/async_tokio quaint quaint/mysql quaint/serde-support serde diesel-async diesel-async/mysql"
 
+      - name: Run Benchmarks (Wtx)
+        if: matrix.backend == 'postgres'
+        run: cargo +stable bench --manifest-path diesel_bench/Cargo.toml --no-default-features --features "postgres tokio/rt-multi-thread wtx"
```

Comment:
Is there any reason not to add this to the postgres job above?

Reply:
Ops, didn't notice. Moved